pkg/commands/workload.go

```diff
@@ -64,6 +65,8 @@ const (
 	AnnotationReservedKey     = "annotations"
 	MavenOverwrittenNoticeMsg = "Maven configuration flags have overwritten values provided by \"--params-yaml\"."
 	WebTypeReservedKey        = "web"
+	MavenFlagWildcard         = "--maven*"
+	LocalPathAndSource        = "--local-path with/without --source-image"
```

Comment:
I think it's important to highlight that source image can come without local path, so that's something we should have in mind.

Reply:
added a comment to address the recommended